ui/src/message_popup/author/question_page.jsx

```diff
@@ -0,0 +1,284 @@
+            <EditingComponents.QuestionText 
+              originalText={this.props.originalQuestion !== undefined ? this.props.originalQuestion.text : undefined}
+              questionText={this.state.questionText}
+              onQuestionTextChange={this.onQuestionTextChange}
+              />
+            <EditingComponents.Students 
+              students={this.state.students}
+              addStudent={this.addStudent}
+              removeStudent={this.removeStudent}
+              />
+            <EditingComponents.Examples 
+              type="Good"
+              examplesText={this.state.goodExamplesText}
+              onExamplesChange={this.onGoodExamplesChange}
+              />
+            <EditingComponents.Examples 
+              type="Bad"
+              examplesText={this.state.badExamplesText}
+              onExamplesChange={this.onBadExamplesChange}
+              />
+            <EditingComponents.LearningObjectives 
+              learningObjective={this.state.learningObjective}
+              onChangeLearningObjective={this.onChangeLearningObjective}
+              />
+            <EditingComponents.Indicators 
+              indicator={this.state.indicator}
+              onChangeIndicator={this.onChangeIndicator}
+              />
+
+            <div style={styles.finalButtonRow}>
+              {this.props.originalQuestion !== undefined &&
+                <div>
+                  <RaisedButton 
+                    style={styles.finalButton}
+                    label="Save"
+                    primary={true}
+                    disabled={!this.saveCheckPassed()}
+                    onTouchTap={this.onSaveEdit}
+                    />
+                  <RaisedButton
+                    style={styles.finalButton}
+                    backgroundColor="#CA2300"
+                    labelStyle={{color: 'white'}}
+                    label="Delete"
+                    onTouchTap={this.openDeleteConfirmation}
+                    />
+                </div>
+              }
+              {this.props.originalQuestion === undefined &&
+                <div>
+                  <RaisedButton
+                    style={styles.finalButton}
+                    label="Create"
+                    primary={true}
+                    disabled={!this.saveCheckPassed()}
+                    onTouchTap={this.onCreate}
+                    />
+                </div>
+              }
+            </div>
+          </VelocityTransitionGroup>
+          <Dialog 
+            open={this.state.deleteConfirmationOpen}
+            actions={[
+              <FlatButton
+                label="Cancel"
+                onTouchTap={this.closeDeleteConfirmation}/>,
+              <FlatButton
+                label="Delete"
+                style={{color: "#CA2300"}}
+                onTouchTap={function(){
+                  this.closeDeleteConfirmation();
+                  this.onDeleteQuestion();
+                }.bind(this)}/>
```

Comment:
I'd recommend splitting this out to a method, it's multiple lines and then you can remove the `bind` as well.  Also, if you use the ES6 arrow syntax for functions, `this` will be bound for you.